Gems/ScriptCanvas/SourceModificationScripts/UpdateSpawnableNodes.py

```diff
@@ -0,0 +1,71 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+# -------------------------------------------------------------------------
+
+import sys
+import shutil
+from pathlib import Path
+import json
+
+def isSpawnableAsset(variableDatum):
+    return (variableDatum.get('$type') == 'SpawnableAsset' or
+        variableDatum.get('scriptCanvasType', {}).get('m_azType') == '{A96A5037-AD0D-43B6-9948-ED63438C4A52}')
+
+fullPath = sys.argv[1]
+
+path = Path(fullPath)
+fileName = path.stem
+pathToFile = path.parent.resolve()
+extension = path.suffix
+
+fileNameBackup = fileName + "_bckup"
+fullBackupPath = (path.parent).joinpath(fileNameBackup).with_suffix(extension)
+
+shutil.copy(fullPath, fullBackupPath)
+print("Backup saved to \'%s\'" % fullBackupPath)
+
+with open(fullPath, 'r') as file :
+    filedata = file.read()
+
+prefab_dom = json.loads(filedata)
+# inspect DOM, do fixup - it'll be stored as a python dict
+
+components = prefab_dom['ClassData']['m_scriptCanvas']['Components']
+
+# replace 'Asset' with 'asset' in SpawnableScriptAssetRef variables
+for componentKey, componentValue in components.items():
+    if 'm_variableData' in componentValue:
+        variableData = componentValue['m_variableData']['m_nameVariableMap']
+        for variable in variableData:
+            variableDatum = variable.get('Value', {}).get('Datum', {})
+            if isSpawnableAsset(variableDatum):
+                asset = variableDatum['value'].pop('Asset')
+                variableDatum['value']['asset'] = asset
+
+filedata = json.dumps(prefab_dom, indent=4)
+
+# finally replace all matching type Uuids and type names
+# SpawnTicketInstance -> EntitySpawnTicket
+filedata = filedata.replace('"$type": "SpawnTicketInstance"', '"$type": "AzFramework::EntitySpawnTicket"')
```

Comment:
repeated below: It would be safer to check the actual json here because the whitespace isn't technically guaranteed (a user could have hand authored / modified a file, however unlikely) but I'd settle for using a regex like `r'"$type":\s*"SpawnTicketInstance"'` to be a bit safer.